### PR TITLE
Quick review of some renaming in prep for adding RS pull into this workflow

### PR DESCRIPTION
**Pull request by steeleb**
Yo, KW!

Here's a mindless PR. I'm just making room to move the remote sensing pull over from our Landsat-C2-SRST repo to satisfy #8 now that we have a functioning pull with new masks. All this really means is that I've renamed a bunch of stuff so that I can plug it in at targets group -b-. 

So, all that should be happening is naming convention changes (from b->c, c->d, ..., e->f) across the board.

The only other thing is the tiniest fix that happens on lines 70, 81, 118... etc in c_historical_RS_data_collation/src/combine_metadata_with_pulls.R where I fix an if statement so I actually get data spit out of this pipeline.

This is unlikely to run as I've probably missed a few renames, so really, this is the most mindless PR ask ever.

You're the best.

### CODE REVIEW

File: c_historical_RS_data_collation/src/combine_metadata_with_pulls.R [hunk 1]
```diff
@@ -67,22 +67,22 @@ combine_metadata_with_pulls <- function(file_prefix, version_identifier, collati
       left_join(., metadata_light) %>% 
       mutate(DSWE = str_sub(source, -28, -24))
     # break out the DSWE 1 data
-    if (nrow(points %>% filter(DSWE == 'DSWE1') > 0)) {
+    if (nrow(points %>% filter(DSWE == 'DSWE1')) > 0) {
```
Comment (kathryn-willi) [inline c_historical_RS_data_collation/src/combine_metadata_with_pulls.R:70]: nice parentheses catches!

File: d_calculate_handoff_coefficients/src/collate_handoff_coefficients.R [hunk 2]
```diff
@@ -10,14 +10,14 @@
 #' 
 collate_handoff_coefficients <- function() {
   #list the files in coefficients
-  files <- list.files("c_calculate_handoff_coefficients/mid",
+  files <- list.files("cdcalculate_handoff_coefficients/mid",
```
Comment (kathryn-willi) [inline d_calculate_handoff_coefficients/src/collate_handoff_coefficients.R]:
```suggestion
  files <- list.files("d_calculate_handoff_coefficients/mid",
```

**Review by kathryn-willi — APPROVED** [review]

File: d_calculate_handoff_coefficients/src/collate_handoff_coefficients.R (hunk 2 above)
Comment (kathryn-willi) [reply d_calculate_handoff_coefficients/src/collate_handoff_coefficients.R]: got one! 😆

Comment (steeleb) [reply d_calculate_handoff_coefficients/src/collate_handoff_coefficients.R]: BAHAHAH that one is egregious. THANK YOU

File: c_historical_RS_data_collation/src/combine_metadata_with_pulls.R (hunk 1 above)
Comment (steeleb) [reply c_historical_RS_data_collation/src/combine_metadata_with_pulls.R:70]: this was after I collated a bunch of data and was like... why is there no data?!

**Comment by steeleb** [comment]
Thank you KW!

**Pull request merged**